tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md

```diff
@@ -46,7 +46,7 @@ In order to deploy applications on `SAP Cloud Foundry` you need to provide the C
 
 Now enter the following commands (in this case for the US region):
 
-```bash
+```bash!!!!
```

Comment:
what's that? 😄

Reply:
I really don't know, I assumed it happened in some Tutorial related formatting, together with the indentation change.
However, I think removing it might be the better choice here.

Reply:
Yeah I guess unless it really has to be there it just looks odd 😄